### PR TITLE
[PL] Updated translation for: Concurrency

### PR DESCRIPTION
**Pull request by bartoszgorka**

### CODE REVIEW

File: pl/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -81,7 +93,10 @@ iex> spawn_link(Example, :explode, [])
 ** (EXIT from #PID<0.57.0>) evaluator process exited with reason: :kaboom
 ```
 
-Czasami nie chcemy by awaria jednego procesu, spowodowała zamknięcie połączonego z nim innego procesu. Dlatego też musimy przechwycić informacje o zamknięciu. Wynikiem tej operacji jest wiadomość w postaci krotki: `{:EXIT, from_pid, reason}`.
+Czasami nie chcemy by awaria jednego procesu, spowodowała zamknięcie połączonego z nim innego procesu.
```
Comment (jangromko) [inline pl/lessons/advanced/concurrency.md:96]:
```suggestion
Czasami nie chcemy by awaria jednego procesu spowodowała zamknięcie połączonego z nim innego procesu.
```

File: pl/lessons/advanced/concurrency.md [hunk 2]
```diff
@@ -104,17 +119,19 @@ Exit reason: kaboom
 
 ### Monitoring
 
-A co jeżeli nie chcemy łączyć procesów, ale chcemy nadal być informowani o awariach? Do tego służy mechanizm monitoringu `spawn_monitor`.  Gdy monitorujemy inny proces z naszego procesu, to gdy otrzymamy wiadomość o jego awarii, nasz proces nie ulegnie awarii ani też nie będziemy musieli jawnie obsłużyć sygnału zamknięcia.
+A co jeżeli nie chcemy łączyć procesów, ale chcemy nadal być informowani o awariach?
```
Comment (jangromko) [inline pl/lessons/advanced/concurrency.md:122]:
```suggestion
A co, jeżeli nie chcemy łączyć procesów, ale chcemy nadal być informowani o awariach?
```

File: pl/lessons/advanced/concurrency.md [hunk 3]
```diff
@@ -126,7 +143,9 @@ Exit reason: kaboom
 
 ## Agenci
 
-Agenci są pewnego rodzaju abstrakcją nad procesami służącą do zarządzania ich stanem. Możemy się do nich odwołać z poziomu innego procesu, aplikacji, albo innego węzła. Aktualny stan agenta jest równy ostatniej zwróconej przez proces wartości:
+Agenci są pewnego rodzaju abstrakcją nad procesami służącą do zarządzania ich stanem w tle.
+Możemy się do nich odwołać z poziomu innego procesu aplikacji, albo innego węzła.
+Aktualny stan agenta jest równy ostatniej zwróconej przez agenta wartości:
```
Comment (jangromko) [inline pl/lessons/advanced/concurrency.md:148]:
```suggestion
Aktualny stan agenta jest równy wartości zwracanej przez naszą funkcję:
```

File: pl/lessons/advanced/concurrency.md [hunk 4]
```diff
@@ -126,7 +143,9 @@ Exit reason: kaboom
 
 ## Agenci
 
-Agenci są pewnego rodzaju abstrakcją nad procesami służącą do zarządzania ich stanem. Możemy się do nich odwołać z poziomu innego procesu, aplikacji, albo innego węzła. Aktualny stan agenta jest równy ostatniej zwróconej przez proces wartości:
+Agenci są pewnego rodzaju abstrakcją nad procesami służącą do zarządzania ich stanem w tle.
+Możemy się do nich odwołać z poziomu innego procesu aplikacji, albo innego węzła.
```
Comment (jangromko) [inline pl/lessons/advanced/concurrency.md:147]:
```suggestion
Możemy się do nich odwołać z poziomu innego procesu aplikacji albo innego węzła.
```

**Review by jangromko — CHANGES_REQUESTED** [review]

**Review by jangromko — APPROVED** [review]
Looks good, thank you for this PR!

**Comment by bartoszgorka** [comment]
@doomspork Is it possible to merge it?

**Pull request merged**